### ISSUE TITLE
Reviewing end of Version 5 - Loops of logic

### ISSUE THREAD

**Issue by bryanmobrien** [gordonmzhu/questions#281]
### What is the associated Watch and Code lesson? 

Write your response here (required).
https://watchandcode.com/courses/60264/lectures/946811
### Include a live demo link so that I can run your code. 

Write your response here (required).
http://plnkr.co/edit/iCIxUGoAuePPGAUYqQfn?p=preview
### Explain the exact steps I need to take to replicate the problem. 

Write your response here (required).

### What did you expect to happen? 

Write your response here (required).

### What actually happened? 
Write your response here (required).
https://gyazo.com/dd101efc12f977d90538d89fd9fe915b

### Describe what you've done so far to troubleshoot the problem. 

Write your response here (required).
As I reviewed the entirety of the todoList javascript program, I noticed something for which I cannot explain.  I made the typo.  I see the typo.  I know the typo exists.  However, the program continues to run.   

In the pnlkr.co link above, I'm focusing on two lines:
line: 20 
`complete: false`   <== complete NOT completed
and line: 34
`todo.completed = !todo.completed;`   <== _complete_**d** 

I can't understand why my code works correctly if I am mixing complete with complete**d**

I would greatly appreciate any insight.

**Comment by gordonmzhu** [comment]
What if you remove line 20 completely? `complete: false` 

What happens?

**Comment by bryanmobrien** [comment]
Okay.  Now I'm even more confused.  I completely removed line 20.
The program still runs as expected.  I'm trying my best to ask great questions, but clearly I am not.  It must be that I'm failing step one of the **Medium** article.   

> How to be great at asking coding questions

Looks like I need to start reviewing the course from the beginning - or - at least at what the completed property of the addTodo method does.

Thank you very much for your response.  Still confused but going to start reviewing the lectures.

**Comment by gordonmzhu** [comment]
Let's break things down into simple parts.

1. In the case that you include `complete: false`, what is the value of `todo.completed`?
2. In the case that you leave out `complete: false`, what is the value of `todo.completed`?

**Comment by bryanmobrien** [comment]
Thank you for the update - I haven't forgotten (Labor Day interrupted).  Hope to revisit this, this evening.

**Comment by bryanmobrien** [comment]
Gordon,

I apologize for the late reply.  I truly appreciate your help on this matter.  Life interrupted me.

I pushed my way through PJS - I couldn’t find a way to go back to the code as it existed in my original question.

Moving forward,  I have a new gameplan:

Take the PJS course, again.  This time, isten for your caveats and abide by your ’suggestions’.

I made good strides the first time through PJS.  I’ll be even more confident when I finish your excellent course, the second time.

Thank you,

Bryan
